elem.go

```diff
@@ -62,6 +62,16 @@ func (t TextNode) Render() string {
 	return string(t)
 }
 
+type CommentNode string
+
+func (c CommentNode) RenderTo(builder *strings.Builder) {
+	builder.WriteString("<!-- " + string(c) + " -->")
+}
+
+func (c CommentNode) Render() string {
```

Comment:
Then we can follow the same pattern as the `Element.Render`:

```go
func (c CommentNode) Render() string {
	var builder strings.Builder
	c.RenderTo(&builder)
	return builder.String()
}
```

Reply:
Fixed.